packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -159,6 +166,15 @@ export default class TabAdvanced extends Component {
             </Col>
           </Row>
         </Container>
+        <h3 className='mt-1 mb-1'>{_('hostMultipathing')}</h3>
+        <ActionButton
+          btnStyle='primary'
+          handler={enableAllHostsMultipathing}
+          handlerParam={hostsDisabledMultipathing}
+          icon='host'
+        >
+          {_('hostEnableMultipathingForAllHost')}
+        </ActionButton>
```

Comment:
Don't we want to also be able to disabled it for all the hosts?

Reply:
It will be a good think if we add it.